packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 42.16.0 - May 17, 2018
+### Fixed
+
+#### salesforce-vscode-apex
+
+* Apex language grammar rules properly categorize Apex syntax, improving code highlighting  ([PR #415](https://github.com/forcedotcom/salesforcedx-vscode/pull/415))
+* Go To Definition and Find All References properly handle custom objects that have a namespace ([PR #413](https://github.com/forcedotcom/salesforcedx-vscode/pull/413))
+
+
+## 42.15.0 - May 10, 2018
```

Comment:
During last week's merge to develop & master branch I missed/skipped the changes to the changelog. Still looking at how that happened.